src/shared/bus-unit-util.c

```diff
@@ -1050,7 +1041,7 @@ static int bus_append_set_credential(sd_bus_message *m, const char *field, const
 
                         r = unbase64mem(p, &decoded, &decoded_size);
                         if (r < 0)
-                                return log_error_errno(r, "Failed to base64 decode encrypted credential: %m");
```

Comment:
this makes it sound like base64 is a encryption algo... just remove the "encrypted" part as it's implied in the field name anyways?

Reply:
btw, can you change the streq() comparison above to check for "Encrypted" suffix?